examples/two-columns-layout/src/plugins/articlesPlugin.tsx

```diff
@@ -0,0 +1,78 @@
+/** @jsx h */
+import {
+  AutocompletePlugin,
+  getAlgoliaResults,
+} from '@algolia/autocomplete-js';
+import { h } from 'preact';
+
+import { ALGOLIA_ARTICLES_INDEX_NAME } from '../constants';
+import { searchClient } from '../searchClient';
+import { ArticleHit } from '../types';
+
+export const articlesPlugin: AutocompletePlugin<ArticleHit, {}> = {
+  getSources({ query }) {
+    if (!query) {
+      return [];
+    }
+
+    return [
+      {
+        sourceId: 'articlesPlugin',
+        getItems() {
+          return getAlgoliaResults<ArticleHit>({
+            searchClient,
+            queries: [
+              {
+                indexName: ALGOLIA_ARTICLES_INDEX_NAME,
+                query,
+                params: {
+                  hitsPerPage: 2,
+                },
+              },
+            ],
+          });
+        },
+        onSelect({ setIsOpen }) {
+          setIsOpen(true);
+        },
+        templates: {
+          header() {
+            return <div className="aa-SourceHeaderTitle">Articles</div>;
+          },
+          item({ item }) {
+            return <ArticleItem hit={item} />;
+          },
+        },
+      },
+    ];
+  },
+};
+
+type ArticleItemProps = {
+  hit: ArticleHit;
+};
+
+const ArticleItem = ({ hit }: ArticleItemProps) => {
+  const articleDate = new Date(hit.date);
+  const articleMonth = articleDate.toLocaleDateString('default', {
```

Comment:
This will avoid mixing languages and have sentences like "Published on févr. 2020".

```suggestion
  const articleMonth = articleDate.toLocaleDateString('en-US', {
```